ros_buildfarm/templates/misc/rosdistro_cache_job.xml.em

```diff
@@ -79,6 +79,9 @@
         ' --cidfile=$WORKSPACE/docker_generate_rosdistro_cache/docker.cid' +
         ' --net=host' +
         ' -v $WORKSPACE/rosdistro_cache:/tmp/rosdistro_cache' +
+        ' -v $HOME/.ssh/known_hosts:/etc/ssh/ssh_known_hosts:ro' +
+        ' -v $SSH_AUTH_SOCK:/tmp/ssh_auth_sock' +
+        ' -e SSH_AUTH_SOCK=/tmp/ssh_auth_sock' +
```

Comment:
I didn't bother making those conditional as there seems to be no harm in them being there if they're not used, but I can change that if it is preferred.